www/source/index.html.slim

```diff
@@ -158,7 +158,7 @@ section.main-section.first
           h4 Isolated Build Environment
           p
             | Define your entire app in a plan file and generate reproducible,
-              trustworthy, versioned builds in a clean-room environment.
+              versioned builds in a clean-room environment.
```

Comment:
Not that they aren't trustworthy 😉 , but this was a bit wordy and it was causing the line to wrap and the remaining bullet features were misaligned.